Source/Scene/ImplicitSubtree.js

```diff
@@ -233,11 +233,12 @@ function parseSubtreeChunks(subtreeView) {
  * isExternal and isActive fields for easier parsing later. This modifies
  * the objects in place.
  *
- * @param {Object[]} bufferHeaders The JSON from subtreeJson.buffers.
+ * @param {Object[]} [bufferHeaders=[]] The JSON from subtreeJson.buffers.
  * @returns {BufferHeader[]} The same array of headers with additional fields.
  * @private
  */
 function preprocessBuffers(bufferHeaders) {
+  bufferHeaders = defaultValue(bufferHeaders, []);
```

Comment:
Nitpick, but this will avoid unnecessary allocations.

```suggestion
  bufferHeaders = defined(bufferHeaders) ? bufferHeaders : [];
```